airbyte-integrations/connectors/source-close-com/source_close_com/source_lc.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from airbyte_cdk.sources.declarative.yaml_declarative_source import YamlDeclarativeSource
+
+"""
+This file provides the necessary constructs to interpret a provided declarative YAML configuration file into
+source connector.
+
+WARNING: Do not modify this file.
+"""
+
+
+# Declarative Source
+class SourceCloseCom(YamlDeclarativeSource):
+    def __init__(self):
+        super().__init__(**{"path_to_yaml": "close_com.yaml"})
```

Comment:
```suggestion
        super().__init__(path_to_yaml="close_com.yaml")
```

Reply:
This is an auto-generated code that no need to touch. This is how all the `Source` classes should look like

Reply:
:shrug: okay